_articles/maintaining-balance-for-open-source-maintainers.md

```diff
@@ -0,0 +1,220 @@
+
+<aside markdown="1" class="pquote">
+  <img src="https://avatars.githubusercontent.com/mansona?s=180" class="pquote-avatar" alt="avatar">
+ I was on a podcast a while ago and we were chatting about open source maintenance and sustainability. I found that even a small number of people supporting my work on GitHub helped me make a quick decision not to sit in front of a game but instead to do one little thing with open source.
+  <p markdown="1" class="pquote-credit">
+— <a href="https://github.com/mansona">@mansona</a>, maintainer of EmberJS
+  </p>
+</aside>
+
+* **Use tools:** Explore tools like [GitHub Copilot](https://github.com/features/copilot/) and [GitHub Actions](https://github.com/features/actions) to automate mundane tasks and free up your time for more meaningful contributions.
+
+<aside markdown="1" class="pquote">
+ Use [Copilot](https://github.com/features/copilot/) for the boring stuff - do the fun stuff
+  <p markdown="1" class="pquote-credit">
+— open source maintainer
+  </p>
+</aside>
+
+* **Rest and recharge:** Make time for your hobbies and interests outside of open source. Take weekends off to unwind and rejuvenate–and set your [GitHub status](https://docs.github.com/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile#setting-a-status) to reflect your availability! A good night's sleep can make a big difference in your ability to sustain your efforts long-term.
+
+  If you find certain aspects of your project particularly enjoyable, try to structure your work so you can experience it throughout your day.
+
+<aside markdown="1" class="pquote">
+  <img src="https://avatars.githubusercontent.com/danielroe?s=180" class="pquote-avatar" alt="avatar">
+ I'm finding more opportunity to sprinkle ‘moments of creativity' in the middle of the day rather than trying to switch off in evening.
+  <p markdown="1" class="pquote-credit">
+— <a href="https://github.com/danielroe">@danielroe</a>, maintainer of Nuxt
+  </p>
+</aside>
+
+* **Set boundaries:** You can't say yes to every request. This can be as simple as saying, "I can't get to that right now and I do not have plans to in the future," or listing out what you're interested in doing and not doing in the README. For instance, you could say: "I only merge PRs which have clearly listed reasons why they were made," or, "I only review issues on alternate Thursdays from 6 -7 pm.”This sets expectations for others, and gives you something to point to at other times to help de-escalate demands from contributors or users on your time.
```

Comment:
[nitpicking] Some maintainers might find it difficult to say no due to having a certain personality type or cultural upbringing, so having a more empathetic tone in this paragraph would give them more courage to set boundaries. 
```suggestion
* **Set boundaries:** Aunt May from *SpiderMan* is onto something when she told Peter Parker "You do too much. You're not SuperMan you know?" We know it's difficult sometimes to say no and that you don't want to offend your contributions, so we came up with ways to help you. For example, you can say something like "I can't get to that right now and I do not have plans to in the future," or "I only merge PRs which have clearly listed reasons why they were made," or, "I only review issues every other Thursday from 6 -7 pm.” You can also list out what you're interested in doing and not doing in the README.
Doing these methods sets expectations for others, and gives you something to point to at other times to help de-escalate demands from contributors or users on your time.
```